modules/dkan/dkan_workflow/modules/dkan_workflow_permissions/dkan_workflow_permissions_sm.js

```diff
@@ -0,0 +1,48 @@
+(function ($) {
```

Comment:
We don't need a whole second JS file just to use a different field ID. Use the Drupal settings object to pass the correct input ID to the script.